src/commands/deploy.yaml

```diff
@@ -0,0 +1,31 @@
+description:
+  Push and deploy the current version
+
+parameters:
+  instance-name:
+    type: string
+    description: "The instace this version will be deployed to"
+  env-name:
+    type: string
+    description: "The environmant containing this instance"
+  image-name:
+    type: string
+    description: "The environmant containing this instance"
```

Comment:
environmant -> environment

Looks like this is meant to have a different description.